### DISCUSSION TITLE
Custom Filter Inputs Dropdown

### DISCUSSION THREAD

**Discussion by thanjeys** [Power-Components/livewire-powergrid#681] [Ideas]
### Have you searched through other issues to see if your problem is already reported or has been fixed?

Yes, I did not find it.

### Did you read the documentation?

Yes, I did not find it.

### Have you tried to publish the views?

Yes - I didn't work.

### Is there an error in the console?

No

### Laravel

8.75

### Livewire

2.5

### Alpine JS

3.0.6

### Theme

Bootstrap

### Describe the bug.

Hello,

Custom Filter Option is required:

For example, If User wants to see Last 1, 2 , 5 year Records means they should select Date from to end. 

How can i make Custom  Dropdown ?  If user select option, Records would have shown

Ex:
https://datatables.yajrabox.com/fluent/advance-filter

Thanks

### To Reproduce...

First click on "FOO" then....

### Extra information

```markdown
<?php
 //...
```

**Comment by luanfreitasdev** [comment]
I didn't quite understand what you need. Can you give more details? I believe it's a feature request
